wpeverest/user-registration
# UR-1660 Tweak - Display ajax error on form save, submit, login and profile edit if ajax somehow has been disabled

### All Submissions:






### Changes proposed in this Pull Request:
Previously when ajax request has been somehow blocked by cache and security plugins like WordFence, then our actions like form save/update. form submit, login and profile edit all were affected and no error message was displayed to the user. Due to this a lot of users have deactivated our plugin thinking that its an error on our plugin side. To mitigate this, this PR has added error messages that error lies within the AJAX request and this hint will help them solve the error.

### How to test the changes in this Pull Request:

1. There is no exact method to block admin ajax action or for some specific action, so to test this we have to edit configuration files like .htaccess to block all admin ajax request, this will affect the whole site, so only verify our mention actions only.
2. Setup you local system to use apache server and edit .htaccess files found in Local Sites/your site/app/public folder and add the following code in between `<IFModule></IFModule>` to the .htaccess file.
```
RewriteCond %{REQUEST_URI} ^/wp-admin/admin-ajax\.php$
RewriteRule ^(.*)$ - [F,L]
```
4.  Test the form save, submit, login and profile edit actions and verify if messages are being displayed properly.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Display ajax error on form save, submit, login and profile edit if ajax somehow has been disabled.
